### DISCUSSION TITLE
What was the last version still using the Apache License?

### DISCUSSION THREAD

**Discussion by ONUHWT** [Megabit/Blazorise#3021] [Q&A] [answered]
What was the last version published to Nuget that still used the Apache 2.0 License?  I have a project that cannot use your new license and want to make sure I do not upgrade beyond the Apache Version.

Thanks

**Comment by stsrki** [comment, marked as answer]
The last "free" version that was released before the license change is 0.9.3.6.

**Reply by ONUHWT** [reply]
thank you
